tests/platform_tests/sfp/conftest.py

```diff
@@ -11,23 +13,52 @@ def teardown_module():
     ans_host.file(path=file_path, state='absent')
 
 
-@pytest.fixture(autouse=True)
-def update_la_ignore_errors_list_for_mlnx(duthost, loganalyzer):
-    extended_ignore_list = []
+def check_pmon_service(duthost):
+    pmon_service_state = duthost.get_service_props("pmon")
+    return pmon_service_state["ActiveState"] == "active"
 
-    if loganalyzer:
-        if duthost.facts["asic_type"] in ["mellanox"]:
-            extended_ignore_list.append("kernel.*Eeprom query failed*")
-            # Ignore PMPE error https://github.com/sonic-net/sonic-buildimage/issues/7163
-            extended_ignore_list.append(r".*ERR pmon#xcvrd: Receive PMPE error event on module.*")
 
-            for host in loganalyzer:
-                loganalyzer[host].ignore_regex.extend(extended_ignore_list)
+def restart_pmon(duthost):
+    duthost.shell("service pmon restart")
+    if not wait_until(1, 60, 0, check_pmon_service, duthost):
+        pytest.fail('pmon service is not up after 60 seconds. Test failed')
+
+
+def modify_daemon_file(daemon_dict):
+    daemon_dict["skip_xcvrd"] = True
+    daemon_dict.pop("delay_xcvrd", None)
+    daemon_dict.pop("skip_xcvrd_cmis_mgr", None)
+    return daemon_dict
```

Comment:
@AharonMalkin The dom thread is still accessing the module even if CMIS mgr is skipped. We should stop the Xcvrd itself

Reply:
Hi @prgeor I think the code will first add "skip_xcvrd" and then remove "delay_xcvrd" and "skip_xcvrd_cmis_mgr", by this it will stop the xcvrd.